src/Microsoft.SqlTools.ServiceLayer/LanguageServices/BindingQueue.cs

```diff
@@ -89,18 +89,12 @@ public bool IsCancelRequested
         /// </summary>
         public virtual QueueItem QueueBindingOperation(
             string key,
-            Func<IBindingContext, CancellationToken, object> bindOperation,
-            Func<IBindingContext, object> timeoutOperation = null,
-            Func<Exception, object> errorHandler = null,
+            Func<IBindingContext, CancellationToken, object?> bindOperation,
+            Func<IBindingContext, object>? timeoutOperation = null,
+            Func<Exception, object>? errorHandler = null,
             int? bindingTimeout = null,
             int? waitForLockTimeout = null)
         {
-            // don't add null operations to the binding queue
```

Comment:
Verified that we're never calling this anywhere that doesn't pass in a bindOperation. As long as no one does something weird and specifically pass in null here we don't need this.